Singer.API/Profiles/EventProfile.cs

```diff
@@ -25,9 +26,13 @@ public EventProfile()
          CreateMap<UpdateEventDTO, Event>()
             .ForMember(x => x.AllowedAgeGroups, opt => opt.MapFrom(src =>
                 src.AllowedAgeGroups.Sum(x => Convert.ToInt32(x))));
+
+         CreateMap<Event, EventDescriptionDTO>()
+            .ForMember(x => x.AgeGroups, opt => opt.MapFrom(src =>
+               ToAgeGroupList(src.AllowedAgeGroups)));
       }
 
-      private List<AgeGroup> ToAgeGroupList(AgeGroup bitmap)
+      public static List<AgeGroup> ToAgeGroupList(AgeGroup bitmap)
```

Comment:
Why was this changed?

Reply:
So I could reuse it in a different class.